com.unity.multiplayer.mlapi/Runtime/Configuration/MLAPIConstants.cs

```diff
@@ -7,9 +7,6 @@ internal static class MLAPIConstants
     {
         internal const string MLAPI_PROTOCOL_VERSION = "13.0.0";
 
-        internal const byte MLAPI_CERTIFICATE_HAIL = 0;
-        internal const byte MLAPI_CERTIFICATE_HAIL_RESPONSE = 1;
-        internal const byte MLAPI_GREETINGS = 2;
         internal const byte MLAPI_CONNECTION_REQUEST = 3;
```

Comment:
Do we want to adjust enum values so that we don't start at 3?

Reply:
yeah, we might but I simply left that decision/discussion for later here.
_(personally, I don't like starting from 0 because I think 0 should be used for empty/null/invalid/default value and not for something meaningful but anyways...)_

Reply:
I agree, I like the convention of mapping "None" to 0.

Reply:
I think at one time I tried making these enum's but I found that even if I declared the enum as a byte type I had to then cast it in all the function calls which made a big mess.  Probably should rewrite the functions that take these to receive an actual (enum) type not a byte